src/main/kotlin/data/Project.kt

```diff
@@ -0,0 +1,9 @@
+package data
+
+data class Project(
+        var id: Int,
+        val targetLanguage: Language,
+        val sourceLanguage: Language,
+        var title: String,
```

Comment:
Originally I thought that we may want to titles for the project but thinking about it now there is no need since its already specific to a book and I doubt our users will want to rename their projects